README.md

```diff
@@ -0,0 +1,31 @@
+# Ubuntu Keyboard
+
+This is a QML and C++ based Keyboard Plugin for Maliit, based on the Maliit Reference plugin, taking into account the special UI/UX requests of Ubuntu Phone.
+
+## Updating presage language models
+
+If the source material used for presage's prediction models is update the databases can be regenerated by running "make lang_db_$lc" within the plugin's "src" directory (where $lc is the language code for the updated language, e.g. "make lang_db_en" for English)
+
+## Updating translations
+
+After building the keyboard (such as with `dpkg-buildpackage -us -uc`), run `make pot` from the `po` directory to rebuild the translation source file.
+
+## On-device testing
+
+To test a pull request on your device, type
+
+    sudo ubports-qa install keyboard-component X
+
+where `X` is the code of a merge request to this repository.
```

Comment:
Nitpick, it's the number rather than the code of the PR. Code may cause confusion because `#x` could be the code of a PR, rather than 'number' which just implies `x`.

Reply:
#88

Reply:
I think that even with its shortcomings, it's worth having these instructions available. Do you agree? If so, I think we're ready.